deploy/charts/rook-ceph/values.yaml

```diff
@@ -63,7 +63,7 @@ csi:
   # Set to true to enable host networking for CSI CephFS and RBD nodeplugins. This may be necessary
   # in some network configurations where the SDN does not provide access to an external cluster or
   # there is significant drop in read/write performance.
-  enableCSIHostNetwork: true
+  enableCSIHostNetwork: false
```

Comment:
This will actually change the default value the chart supplies for all Helm users. Is it safe to change this and cause users' CSI pods to be redeployed?

Reply:
@BlaineEXE still figuring out the impact of this on upgraded cluster